content/docs/1.5.0/references/networking.md

```diff
@@ -9,10 +9,12 @@ This page documents the networking communication between components in the Longh
 to control the inbound/outbound traffic to/from Longhorn components. This helps to reduce the damage when a malicious pod breaks into the in-cluster network.
 
 We have provided some NetworkPolicy example yamls at [here](https://github.com/longhorn/longhorn/tree/master/examples/network-policy).
+Or you can enable the setting in the helm chart to install these NetworkPolicy [https://github.com/longhorn/longhorn/blob/master/chart/values.yaml]
 Note that depending on the deployed [CNI](https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/), not all Kubernetes clusters support NetworkPolicy.
 See [here](https://kubernetes.io/docs/concepts/services-networking/network-policies/) for more detail.
 
 > Note: If you are writing network policies, please revisit this page before upgrading Longhorn to make the necessary adjustments to your network policies.
+> Note: With network policies enabled, it may fail the recurring job since there might have some delay when Kubernetes applying netowk policies to the pod created by the job.
```

Comment:
Should mention it would depend on the ingress controller stability, and it's actually beyond the longhorn control.

Reply:
sure

Reply:
Fixed!